### ISSUE TITLE
QGIS crashes on Gnome Wayland

### ISSUE THREAD

**Issue by jecogeo** [qgis/QGIS#34781]
QGIS doesn't start in Arch Linux Gnome Wayland (all fully updated) since v 3.8.3-2. Later versions returns:

```
Info: Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
QGIS died on signal 11[1]    12181 abort (core dumped)  qgis.
```

I've tried `QT_QPA_PLATFORM=xcb qgis` to force X11 and the same occurs. Downgrading to 3.8.3-2 works fine.

My journal output is useless for me, can't figure out nothing from it.

```
Feb 28 16:38:09 HAL9000 systemd[1]: Started Process Core Dump (PID 9660/UID 0).
Feb 28 16:38:09 HAL9000 systemd-coredump[9661]: Process 9654 (qgis) of user 1000 dumped core.
                                                
                                                Stack trace of thread 9654:
                                                #0  0x00007f05e972cce5 raise (libc.so.6 + 0x3bce5)
                                                #1  0x00007f05e9716857 abort (libc.so.6 + 0x25857)
                                                #2  0x000055af514e322c n/a (qgis + 0xe22c)
                                                #3  0x00007f05e972cd70 __restore_rt (libc.so.6 + 0x3bd70)
                                                #4  0x00007f05eb60af3c _ZN6QColor6setRgbEiiii (libQt5Gui.so.5 + 0x317f3c)
                                                #5  0x00007f05ed8ef09a call_init.part.0 (ld-linux-x86-64.so.2 + 0x1109a)
                                                #6  0x00007f05ed8ef1a1 _dl_init (ld-linux-x86-64.so.2 + 0x111a1)
                                                #7  0x00007f05e982a905 _dl_catch_exception (libc.so.6 + 0x139905)
                                                #8  0x00007f05ed8f30e8 dl_open_worker (ld-linux-x86-64.so.2 + 0x150e8)
                                                #9  0x00007f05e982a8a8 _dl_catch_exception (libc.so.6 + 0x1398a8)
                                                #10 0x00007f05ed8f296e _dl_open (ld-linux-x86-64.so.2 + 0x1496e)
                                                #11 0x00007f05e245334c n/a (libdl.so.2 + 0x134c)
                                                #12 0x00007f05e982a8a8 _dl_catch_exception (libc.so.6 + 0x1398a8)
                                                #13 0x00007f05e982a973 _dl_catch_error (libc.so.6 + 0x139973)
                                                #14 0x00007f05e2453ab9 n/a (libdl.so.2 + 0x1ab9)
                                                #15 0x00007f05e24533da dlopen (libdl.so.2 + 0x13da)
                                                #16 0x00007f05eb05b761 n/a (libQt5Core.so.5 + 0x2a1761)
                                                #17 0x00007f05eb055429 _ZN8QLibrary4loadEv (libQt5Core.so.5 + 0x29b429)
                                                #18 0x00007f05ea4f2b31 _ZN19QgsProviderRegistry4initEv (libqgis_core.so.3.12.0 + 0xa38b31)
                                                #19 0x00007f05ea4f382b _ZN19QgsProviderRegistryC2ERK7QString (libqgis_core.so.3.12.0 + 0xa3982b)
                                                #20 0x00007f05ea4f393a _ZN19QgsProviderRegistry8instanceERK7QString (libqgis_core.so.3.12.0 + 0xa3993a)
                                                #21 0x00007f05ea265a37 _ZN14QgsApplication8initQgisEv (libqgis_core.so.3.12.0 + 0x7aba37)
                                                #22 0x00007f05ed0c11fb _ZN7QgisAppC1EP13QSplashScreenbbRK7QStringS4_P7QWidget6QFlagsIN2Qt10WindowTypeEE (libqgis_app.so.3.12.0 + 0x2861fb)
                                                #23 0x000055af514df660 n/a (qgis + 0xa660)
                                                #24 0x00007f05e9718023 __libc_start_main (libc.so.6 + 0x27023)
                                                #25 0x000055af514e26ce _start (qgis + 0xd6ce)
                                                
                                                Stack trace of thread 9655:
                                                #0  0x00007f05ec04bcf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                                                #1  0x00007f05d80b9e1c n/a (i965_dri.so + 0x4f8e1c)
                                                #2  0x00007f05d80b9a18 n/a (i965_dri.so + 0x4f8a18)
                                                #3  0x00007f05ec04546f start_thread (libpthread.so.0 + 0x946f)
                                                #4  0x00007f05e97f03d3 __clone (libc.so.6 + 0xff3d3)
                                                
                                                Stack trace of thread 9656:
                                                #0  0x00007f05ec04bcf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                                                #1  0x00007f05d80b9e1c n/a (i965_dri.so + 0x4f8e1c)
                                                #2  0x00007f05d80b9a18 n/a (i965_dri.so + 0x4f8a18)
                                                #3  0x00007f05ec04546f start_thread (libpthread.so.0 + 0x946f)
                                                #4  0x00007f05e97f03d3 __clone (libc.so.6 + 0xff3d3)
                                                
                                                Stack trace of thread 9657:
                                                #0  0x00007f05ec04bcf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                                                #1  0x00007f05d80b9e1c n/a (i965_dri.so + 0x4f8e1c)
                                                #2  0x00007f05d80b9a18 n/a (i965_dri.so + 0x4f8a18)
                                                #3  0x00007f05ec04546f start_thread (libpthread.so.0 + 0x946f)
                                                #4  0x00007f05e97f03d3 __clone (libc.so.6 + 0xff3d3)
                                                
                                                Stack trace of thread 9658:
                                                #0  0x00007f05ec04bcf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                                                #1  0x00007f05d80b9e1c n/a (i965_dri.so + 0x4f8e1c)
                                                #2  0x00007f05d80b9a18 n/a (i965_dri.so + 0x4f8a18)
                                                #3  0x00007f05ec04546f start_thread (libpthread.so.0 + 0x946f)
                                                #4  0x00007f05e97f03d3 __clone (libc.so.6 + 0xff3d3)
                                                
                                                Stack trace of thread 9659:
                                                #0  0x00007f05e97e5abf __poll (libc.so.6 + 0xf4abf)
                                                #1  0x00007f05e2269240 n/a (libglib-2.0.so.0 + 0x6c240)
                                                #2  0x00007f05e2269311 g_main_context_iteration (libglib-2.0.so.0 + 0x6c311)
                                                #3  0x00007f05eb0b8020 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2fe020)
                                                #4  0x00007f05eb05e4cc _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2a44cc)
                                                #5  0x00007f05eae82e52 _ZN7QThread4execEv (libQt5Core.so.5 + 0xc8e52)
                                                #6  0x00007f05de662b28 n/a (libQt5DBus.so.5 + 0x15b28)
                                                #7  0x00007f05eae83fc6 n/a (libQt5Core.so.5 + 0xc9fc6)
                                                #8  0x00007f05ec04546f start_thread (libpthread.so.0 + 0x946f)
                                                #9  0x00007f05e97f03d3 __clone (libc.so.6 + 0xff3d3)
Feb 28 16:38:09 HAL9000 systemd[1]: systemd-coredump@4-9660-0.service: Succeeded.
```
System details:

```
OS: Arch Linux x86_64 
Host: Inspiron 5480 
DE: GNOME 3.34.4 
CPU: Intel i7-8565U (8) @ 4.600GHz 
GPU: Intel UHD Graphics 620 
Memory: 15794MiB 
```

**Comment by roya0045** [comment]
Have you retested with 3.12?

**Comment by jecogeo** [comment]
Yes, Tested and can confirm its solved in QGIS 3.12. Versions from 3.8 until 3.11 dos not work. Thanks
